pkg/configtx/application.go

```diff
@@ -157,6 +152,7 @@ func (c *ConfigTx) RemoveAnchorPeer(orgName string, anchorPeerToRemove Address)
 }
 
 // AddACLs adds ACLS to an existing channel config application.
+// Adding ACLs infers the application ConfigGroup exists.
```

Comment:
If it doesn't exist, this will panic, correct? I think we should document that.

Reply:
I documented the panic for each removal function